<ui version="4.0" >
 <class>MainDialogBase</class>
 <widget class="QDialog" name="MainDialogBase" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>502</width>
    <height>468</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle" >
   <string>MainWindow</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2" >
   <property name="spacing" >
    <number>2</number>
   </property>
   <property name="margin" >
    <number>1</number>
   </property>
   <item>
    <widget class="QLabel" name="mImageLabel" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Maximum" hsizetype="Maximum" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text" >
      <string/>
     </property>
     <property name="pixmap" >
      <pixmap resource="../assburner.qrc" >:/images/AssburnerLogo.png</pixmap>
     </property>
     <property name="alignment" >
      <set>Qt::AlignTop</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="spacing" >
      <number>6</number>
     </property>
     <property name="margin" >
      <number>0</number>
     </property>
     <item>
      <layout class="QVBoxLayout" >
       <property name="spacing" >
        <number>6</number>
       </property>
       <property name="margin" >
        <number>0</number>
       </property>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType" >
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>17</width>
           <height>24</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="mShowAssignmentsButton" >
         <property name="maximumSize" >
          <size>
           <width>25</width>
           <height>25</height>
          </size>
         </property>
         <property name="text" >
          <string>+</string>
         </property>
         <property name="checkable" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" >
       <property name="spacing" >
        <number>6</number>
       </property>
       <property name="margin" >
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="mStatusLabel" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Preferred" hsizetype="MinimumExpanding" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize" >
          <size>
           <width>250</width>
           <height>23</height>
          </size>
         </property>
         <property name="font" >
          <font>
           <family>Arial</family>
           <pointsize>12</pointsize>
           <weight>50</weight>
           <italic>false</italic>
           <bold>false</bold>
           <underline>false</underline>
           <strikeout>false</strikeout>
          </font>
         </property>
         <property name="text" >
          <string>&lt;p align="center">Status [ &lt;font size="+1">Ready&lt;/font> ]&lt;/p></string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" >
         <property name="spacing" >
          <number>6</number>
         </property>
         <property name="margin" >
          <number>0</number>
         </property>
         <item>
          <widget class="QPushButton" name="mAFButton" >
           <property name="sizePolicy" >
            <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize" >
            <size>
             <width>79</width>
             <height>31</height>
            </size>
           </property>
           <property name="toolTip" >
            <string>Start Burner Queue Viewer</string>
           </property>
           <property name="text" >
            <string/>
           </property>
           <property name="icon" >
            <iconset>
             <normaloff>:images/assfreezer_button.png</normaloff>:images/assfreezer_button.png</iconset>
           </property>
           <property name="iconSize" >
            <size>
             <width>79</width>
             <height>30</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="ClientLogButton" >
           <property name="sizePolicy" >
            <sizepolicy vsizetype="Fixed" hsizetype="Minimum" >
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize" >
            <size>
             <width>79</width>
             <height>31</height>
            </size>
           </property>
           <property name="text" >
            <string>Log</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="OptionsButton" >
           <property name="maximumSize" >
            <size>
             <width>79</width>
             <height>31</height>
            </size>
           </property>
           <property name="text" >
            <string>Options</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="DisableButton" >
       <property name="enabled" >
        <bool>true</bool>
       </property>
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize" >
        <size>
         <width>200</width>
         <height>61</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>200</width>
         <height>61</height>
        </size>
       </property>
       <property name="font" >
        <font>
         <family>Arial</family>
         <pointsize>20</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
         <underline>false</underline>
         <strikeout>false</strikeout>
        </font>
       </property>
       <property name="text" >
        <string>Stop The Burn</string>
       </property>
       <property name="default" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="mActiveAssignmentsGroup" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Maximum" hsizetype="Preferred" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title" >
      <string>Assigned Jobs</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout" >
      <property name="margin" >
       <number>4</number>
      </property>
      <item>
       <widget class="RecordTreeView" name="mAssignmentsView" />
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>RecordTreeView</class>
   <extends>QTreeView</extends>
   <header>recordtreeview.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../assburner.qrc" />
  <include location="../assburner.qrc" />
  <include location="../assburner.qrc" />
 </resources>
 <connections/>
</ui>
